internal/parser/node_groups.go

```diff
@@ -5,17 +5,20 @@ import (
 	"strings"
 
 	logger "github.com/gookit/slog"
+	"github.com/stg-tud/bp2022_netlab/internal/customtypes"
 	"github.com/stg-tud/bp2022_netlab/internal/experiment"
 	"github.com/stg-tud/bp2022_netlab/internal/movementpatterns"
 )
 
 // Input format of a NodeGroup configuration
 type inputNodeGroup struct {
-	Prefix        any `required:"true"`
-	NoNodes       any `default:"1"`
-	MovementModel any `default:"Static"`
-	NodesType     any `default:"PC"`
-	Networks      []string
+	Prefix             any `required:"true"`
+	NoNodes            any `default:"1"`
+	MovementModel      any `default:"Static"`
+	NodesType          any `default:"PC"`
+	Networks           []string
+	PredefinedPosition bool
```

Comment:
Is this parameter really necessary? If the user doesn't set a position, this value should be false, if he does, it should be true. It would be redundand to use another bool for that. Maybe have a look at the `any` + default values [here](https://github.com/stg-tud/bp2022_netlab/blob/main/CONTRIBUTING.md#parser), but there are other approaches as well.